cli/js/web/console.ts

```diff
@@ -603,18 +650,21 @@ function createObjectString(
     return createTypedArrayString(
       Object.getPrototypeOf(value).constructor.name,
       value,
-      ...args
+      consoleContext,
+      level,
+      inspectOptions
     );
   } else {
     // Otherwise, default object formatting
-    return createRawObjectString(value, ...args);
+    return createRawObjectString(value, consoleContext, level, inspectOptions);
   }
 }
 
 export function stringifyArgs(
   args: unknown[],
-  { depth = DEFAULT_MAX_DEPTH, indentLevel = 0 }: InspectOptions = {}
+  inspectOptions: InspectOptions = {}
 ): string {
+  const rInspectOptions = { ...DEFAULT_INSPECT_OPTIONS, ...inspectOptions };
```

Comment:
What does `r` means in `rInspectOptions`?

Reply:
required

Reply:
Why don't name it `requiredInspectOptions` instead?

Reply:
🤷‍♂️